src/App.test.js

```diff
@@ -1,8 +0,0 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
-
```

Comment:
Please add unit tests for components of the app and also for some pure functions in Redux.
Please add integration tests using `React Testing Library`

Reply:
I completely forgot the tests. Sorry, will add them right away.